### DISCUSSION TITLE
How to lossless convert gif to webp

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [dlemstra/Magick.NET#964] [Help] [answered]
Hello. I have tried the below code and looks like working

However I want to convert gif to webp with lossless quality and best compression (cpu time etc not important)

How can I do that?

Thank you very much

```
            using (var images = new MagickImageCollection(imagePath))
            {
                images.Write("gg.webp");
            }
```

**Comment by dlemstra** [comment, marked as answer]
You can use the `WebPWriteDefines` class to set various options for encoding a `WebP` image.
